src/main/java/graphql/schema/GraphqlDirectivesContainerTypeBuilder.java

```diff
@@ -31,17 +57,32 @@ public B withAppliedDirectives(GraphQLAppliedDirective... directives) {
         return (B) this;
     }
 
+    /**
+     * @param directive the directive to add
+     *
+     * @return this builder
+     *
+     * @deprecated - use {@link #addAppliedDirective(GraphQLAppliedDirective)} instead
+     */
+    @Deprecated(since = "2025-02-16", forRemoval = true)
     public B withAppliedDirective(GraphQLAppliedDirective directive) {
         assertNotNull(directive, () -> "directive can't be null");
         this.appliedDirectives.add(directive);
         return (B) this;
     }
```

Comment:
I think this method is fine, but it would be confusing to have the plural method start with "add" and the singular one start with a different word, so I gave this a corresponding new method `addAppliedDirective`